frontstage/controllers/party_controller.py

```diff
@@ -262,10 +267,17 @@ def get_survey_list_details_for_party(party_id, tag, business_party_id, survey_i
                 'period': collection_exercise['userDescription'],
                 'submit_by': collection_exercise['events']['return_by']['date'],
                 'collection_exercise_ref': collection_exercise['exerciseRef'],
-                'added_survey': added_survey
+                'added_survey': added_survey,
+                'display_access_button': display_access_button
             }
 
 
+def should_display_access_button(status, ci_type):
```

Comment:
Might make more sense to change to:

```python
def is_closed(status, ci_type):
    return ci_type == 'EQ' or status in CLOSED_STATE
```

Reply:
is_closed is probably not the best name for this function because a seft survey "is closed" but it shouldn't return True because Seft surveys can still be accessed when the survey has been completed. Also that function would return True for all EQ surveys, whether they are closed or not.